lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative 'menu'
+require 'date'
+require 'twilio-ruby'
+
+class Order
+
+  attr_reader :final_order
+
```

Comment:
Do you now know how to obfuscate these credentials?   If not, it's worth looking at how you can use environment variables.